pkg/lang/golang/aws_runtime/Lambda_Dockerfile

```diff
@@ -2,12 +2,10 @@ FROM golang:1.20 as builder
 
 WORKDIR /usr/src/app
 ENV GOOS=linux GOARCH=amd64 CGO_ENABLED=0
-COPY go.mod ./
```

Comment:
Just to check (since I know you have a bunch of changes in flight), did you meant to commit these changes into this PR?

Reply:
yeah the dockerfile changes are necessary to get this to work